<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainClass</class>
 <widget class="QMainWindow" name="QMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1965</width>
    <height>1246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMain</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1965</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QMain.qrc"/>
 </resources>
 <connections/>
</ui>
